react-form-controls/src/index.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+
+class RegistrationForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleUsernameChange = this.handleUsernameChange.bind(this);
+    this.handlePasswordChange = this.handlePasswordChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.state = { username: '' };
+    this.state = { password: '' };
```

Comment:
This won't work, state will just be whatever comes last. You want to add both properties to a single object:
```JAVASCRIPT
this.state = {
  username: '',
  password: ''
};
```